src/core/lib/transport/metadata_batch.h

```diff
@@ -104,6 +112,20 @@ grpc_error* grpc_metadata_batch_add_head(
     grpc_metadata_batch* batch, grpc_linked_mdelem* storage,
     grpc_mdelem elem_to_add) GRPC_MUST_USE_RESULT;
 
+inline grpc_error* GRPC_MUST_USE_RESULT grpc_metadata_batch_add_head(
+    grpc_metadata_batch* batch, grpc_linked_mdelem* storage,
+    grpc_metadata_batch_callouts_index idx) {
+  return grpc_metadata_batch_link_head(batch, storage, idx);
```

Comment:
I might be missing something here, but why do we need this grpc_metadata_batch_add_head function when it seems to be identical to grpc_metadata_batch_link_head?

Reply:
No idea why they both exist, tbh. They're basically identical. I'll ask @roth if there's any particular reason they may both still be there.

For now, my thought was to leave it in place to not rock things too much.

Reply:
A TODO is fine or just following up with roth@ when he gets back.